.circleci/config.yml

```diff
@@ -2,10 +2,10 @@
 version: 2.1
 
 executors:
-  # osx-arm64:
-  #   macos:
-  #     xcode: 14.2.0 # indicate your selected version of Xcode
-  #   resource_class: macos.m1.large.gen1
+  osx-arm64:
+    macos:
+      xcode: 15.0.0
```

Comment:
I chose 15.0.0 because that matches what was last used for osx-64 bulk on GitHub Actions. https://circleci.com/docs/using-macos/

Reply:
That page says:
```
Config          Xcode Version                	macOS Version
15.0.0          Xcode 15 (15A240d)              13.5.1
``

Github Actions provides `macos-14`.
I might be wrong but it looks like  `13.5.1` vs `14.x`

Reply:
Some past discussions imply that it doesn't make too much difference because of the setup done by `run_conda_forge_build_setup`. So leaving at `15.4.0` and we can always downgrade if needed.